### ISSUE TITLE
Privacy Badger is reporting a "ghost tracker."

### ISSUE THREAD

**Issue by sydeburns** [EFForg/privacybadgerfirefox#512]
I keep getting reports that Privacy Badger is blocking fetishmovies.com, but when I check the page's source, there's nothing about that site.  This has just started, and here's an example of a page that shows this: http://www.nytimes.com/2015/08/16/opinion/sunday/how-california-is-winning-the-drought.html?_r=0

**Comment by sydeburns** [comment]
More info: I just opened up the manual for my printer, which resides on my hard disk, and Privacy Badger reported a large number of trackers, just as above.  And, I just checked and it shows the same 29 trackers on this page!  Clearly, something has gone wrong, but I don't know, yet, what it is.  If it helps, I haven't needed to restart Firefox since I reported this.

**Comment by cooperq** [comment]
I don't think this is a bug with privacy badger. It sounds like you might have some malware though...

**Comment by sydeburns** [comment]
On 08/17/2015 09:39 AM, Cooper Quintin wrote:
> I don't think this is a bug with privacy badger. It sounds like you
> might have some malware though...
>

How much malware do you know of that infects Fedora Linux with SELinux 
enabled?

**Comment by cooperq** [comment]
:) okay not much, I was just going based on the fact that nytimes.com is definitely not loading fetishmovies.com.

Anyway, if you restart firefox and open up that same document do you still see the list of trackers? 

Can you also tell me what version of firefox and what version of privacy badger you are running?

**Comment by sydeburns** [comment]
On 08/17/2015 09:53 AM, Cooper Quintin wrote:
> Anyway, if you restart firefox and open up that same document do you
> still see the list of trackers?

By this morning, it was showing the same trackers (over 40) for 
everyplace; restarting Firefox cleared it, so if it's a bug, it's 
probably some sort of garbage collection issue, or reusing an array 
without clearing it out all the time.  I'm using version 1.0.1, and it 
doesn't show the number of trackers by the icon as I've got it set.

**Comment by cooperq** [comment]
Do you by any chance have e10s turned on in firefox? See this issue for more info #208

**Comment by sydeburns** [comment]
On 08/17/2015 10:13 AM, Cooper Quintin wrote:
> Do you by any chance have e10s turned on in firefox? See this issue for
> more info #208 <https://github.com/EFForg/privacybadgerfirefox/issues/208>
>

No.

**Comment by cooperq** [comment]
What version of firefox do you have?

**Comment by sydeburns** [comment]
On 08/17/2015 10:46 AM, Cooper Quintin wrote:
> What version of firefox do you have?
>

34.0 because I'm stuck, for the time being with Fedora 19.  I have to 
make the time to replace my power supply because it's causing occasional 
reboots and I really don't want that to happen during an upgrade.  I 
have a replacement, I just have to put it in, but I'm not really a 
hardware geek so it's a matter of getting tired enough of the situation 
to deal with it.

**Comment by cooperq** [comment]
Can you open up the browser console with ctrl+shift+j and tell me if there are any privacy badger related errors in there?

**Comment by sydeburns** [comment]
On 08/17/2015 03:02 PM, Cooper Quintin wrote:
> Can you open up the browser console with ctrl+shift+j and tell me if
> there are any privacy badger related errors in there?
>

Not since I restarted Firefox.  If this comes up again, I'll check and 
let you know of any.
